### ISSUE TITLE
Problem to wait for an alert with WebDriverExpectedCondition::alertIsPresent()

### ISSUE THREAD

**Issue by Xoto1162** [php-webdriver/php-webdriver#813]
### What are you trying to achieve? (Expected behavior)
Wait for an alert to be visible

### What do you get instead? (Actual behavior)
Facebook\WebDriver\Exception\NoSuchAlertException: no such alert

### How could the issue be reproduced? (Steps to reproduce)
I use this code to wait for an alert to be visible but I immediately receive a NoSuchAlertException

```php
$this->_webDriver->wait()->until(
    WebDriverExpectedCondition::alertIsPresent()
);
```

### Details

* Php-webdriver version: 1.8.2
* PHP version: PHP 7.4.7
* Selenium server version: ChromeDriver 84.0.4147.30
* Operating system: Windows 10
* Browser used + version: 84.0.4147.105 (Build officiel) (64 bits)

**Comment by Xoto1162** [comment]
I investigate the source code and I found the problem

In the file WebDriverExpectedCondition.php you define the function alertIsPresent() but in this function you catch the wrong exception

# Your version
```php
public static function alertIsPresent()
    {
        return new static(
            function (WebDriver $driver) {
                try {
                    var_dump('coucou1');
                    // Unlike the Java code, we get a WebDriverAlert object regardless
                    // of whether there is an alert.  Calling getText() will throw
                    // an exception if it is not really there.
                    $alert = $driver->switchTo()->alert();
                    $alert->getText();

                    return $alert;
                } catch (NoAlertOpenException $e) {
                    var_dump('coucou2');
                    return null;
                }
            }
        );
    }
```

# Correct version

You need to add the use of Facebook\WebDriver\Exception\NoSuchAlertException at the top of the file

```php
public static function alertIsPresent()
    {
        return new static(
            function (WebDriver $driver) {
                try {
                    var_dump('coucou1');
                    // Unlike the Java code, we get a WebDriverAlert object regardless
                    // of whether there is an alert.  Calling getText() will throw
                    // an exception if it is not really there.
                    $alert = $driver->switchTo()->alert();
                    $alert->getText();

                    return $alert;
                } catch (NoSuchAlertException $e) {
                    var_dump('coucou2');
                    return null;
                }
            }
        );
    }
```

**Comment by OndraM** [comment]
Hi, this is already fixed in `main` branch (but was not yet released), see #792 .

You can use `dev-main` as a version in your composer.json temporarily, or wait few days until we tag the next release.
